arcgis-android-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/bookmark/BookmarkView.kt

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright 2019 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.toolkit.bookmark
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.esri.arcgisruntime.mapping.Bookmark
+import com.esri.arcgisruntime.mapping.BookmarkList
+import com.esri.arcgisruntime.toolkit.BR
+import com.esri.arcgisruntime.toolkit.R
+import kotlinx.android.synthetic.main.layout_bookmarkview.view.bookmarkRecyclerView
+
+/**
+ * The BookmarkView will display a list of bookmarks in a [RecyclerView] and allows the user to
+ * select a bookmark and perform some action.
+ *
+ * @since 100.7.0
+ */
+class BookmarkView : FrameLayout {
+
+    private val bookmarksAdapter: BookmarkAdapter = BookmarkAdapter(
+        object : OnItemClickListener<Bookmark> {
+            override fun onItemClick(item: Bookmark) {
+                onItemClickListener?.onItemClick(item)
+            }
+        })
+
+    var bookmarkList: BookmarkList? = null
```

Comment:
I suggest we call the property `bookmarks` instead of `bookmarkList`

Reply:
updated